src/connectors/hits/connectHits.js

```diff
@@ -3,6 +3,7 @@ import {
   checkRendering,
   createDocumentationMessageGenerator,
 } from '../../lib/utils';
+import { addAbsolutePosition } from '../../lib/utils/hits-absolute-position';
```

Comment:
If we follow the above pattern, we can export `addAbsolutePosition` in `utils.ts` and import the function from there.